x-pack/plugins/security_solution/public/detections/components/alerts_table/timeline_actions/use_investigate_in_timeline.tsx

```diff
@@ -108,7 +108,7 @@ export const useInvestigateInTimeline = ({
         <EuiContextMenuItem
           key="investigate-in-timeline-action-item"
           data-test-subj="investigate-in-timeline-action-item"
-          disabled={isFetchingAlertEcs === true}
+          disabled={ecsRowData == null && isFetchingAlertEcs === true}
```

Comment:
ran into a race condition here where `ecsRowData` becomes truthy before the `isFetchingAlertEcs` call is complete